src/js/tech/chromecast.js

```diff
@@ -23,8 +23,19 @@ class Chromecast extends Tech {
         this.apiSession = this.options_.source.apiSession;
         this.receiver = this.apiSession.receiver.friendlyName;
 
-        this.apiMedia.addUpdateListener(::this.onMediaStatusUpdate);
-        this.apiSession.addUpdateListener(::this.onSessionUpdate);
+        let mediaStatusUpdateHandler = ::this.onMediaStatusUpdate;
+        let sessionUpdateHanlder = ::this.onSessionUpdate;
+
+        this.apiMedia.addUpdateListener(mediaStatusUpdateHandler);
+        this.apiSession.addUpdateListener(sessionUpdateHanlder);
+
+        this.on('dispose', () => {
```

Comment:
Oh, I see. This is my first time seeing the bind syntax in the wild. Yeah, seems like this makes it a bit cleaner.